src/System.Management.Automation/engine/parser/Compiler.cs

```diff
@@ -1357,7 +1377,7 @@ internal static Attribute GetAttribute(AttributeAst attributeAst)
             if (attributeType == null)
             {
                 throw InterpreterError.NewInterpreterException(attributeAst, typeof(RuntimeException), attributeAst.Extent,
-                    "CustomAttributeTypeNotFound", ParserStrings.CustomAttributeTypeNotFound, attributeAst.TypeName.FullName);
+                    "TypeNotFound", ParserStrings.TypeNotFound, attributeAst.TypeName.FullName);
```

Comment:
> I think I'd reuse TypeNotFound instead of adding a new message.

But `CustomAttributeTypeNotFound` is not a new resource string, and it looks to me fit pretty well in the `if (attributeType == null)` situation:
`Cannot find the type for custom attribute '{0}'. Make sure that the assembly that contains this type is loaded.`